src/clojure/core.clj

```diff
@@ -8,9 +8,12 @@
 (defn apply [f args]
   (lexical-eval (concat (list f) args)))
 
-(defn println [& more]
+(defn print [& more]
   (print-string (apply str more)))
 
+(defn println [& more]
+  (println-string (apply str more)))
```

Comment:
However this one shouldn't need a new rust primitive, unless there's another reason for it (for instance, if you are trying to use `println` because it manages which cross-platform newline to spit out, I'd understand,  although in this case I just checked it and it should always spit out `\n`) so it should end up something like 

```clojure
(defn println [& more]
      (print-string (str (apply str more) "\n")))
```
Its slightly uglier (which could probably be 'adjusted' but wouldn't be worth it) but this way we don't need another primitive (that itself just adds an `"\n"`. As I'll always say, though, if there's something more to this let me know)

For anyone else reading this:  this is the eventual goal for printing: https://github.com/clojure/clojure/blob/clojure-1.10.1/src/clj/clojure/core.clj#L3733   (although we are some ways from there -- we will need to expose streams, for instance, to Clojure RS)

Although, funnily enough,  clojure *does* have just a single function `newline` all about getting the right newline for the right platform